d800/d800-vendor.mk

```diff
@@ -14,4 +14,20 @@
 
 # This file is generated by device/lge/d800/setup-makefiles.sh
 
-$(call inherit-product, vendor/lge/d800/d800-vendor-blobs.mk)
```

Comment:
Probably want to remove the vendor-blobs makefiles since they don't seem to be referenced anymore.